### ISSUE TITLE
Bundler could not find compatible versions for gem "rack"

### ISSUE THREAD

**Issue by Spectator6** [castwide/solargraph#6]
Upon start, Visual Studio Code notifies me to install the solargraph gem. Whenever I include it in my gemfile and run "bundle", I receive the following error message:

```
Bundler could not find compatible versions for gem "rack":
  In Gemfile:
    rails (~> 5.0) was resolved to 5.0.0, which depends on
      actionpack (= 5.0.0) was resolved to 5.0.0, which depends on
        rack (~> 2.0)

    better_errors was resolved to 2.3.0, which depends on
      rack (>= 0.9.0)

    rails (~> 5.0) was resolved to 5.0.0, which depends on
      actionpack (= 5.0.0) was resolved to 5.0.0, which depends on
        rack-test (~> 0.6.3) was resolved to 0.6.3, which depends on
          rack (>= 1.0)

    solargraph was resolved to 0.1.0, which depends on
      sinatra (~> 1.4) was resolved to 1.4.0, which depends on
        rack (>= 1.5.2, ~> 1.5)

    sass-rails (~> 5.0.6) was resolved to 5.0.6, which depends on
      sprockets (< 4.0, >= 2.8) was resolved to 3.7.1, which depends on
        rack (< 3, > 1)
```
Any ideas on how to resolve this? Please let me know if you need any more information, thank you!

**Comment by castwide** [comment]
Solargraph 0.10 depends on Rack 1, while Rails 5 requires Rack 2. Solargraph 0.11 includes a dependency upgrade that resolves the issue. I tested it with a Rails 5 codebase yesterday and was able to install the bundle without conflicts.

I went ahead and published version 0.11 this afternoon. If you still encounter problems, please let me know.

**Comment by Spectator6** [comment]
Thank you so much for your quick turnaround, version 0.11 fixed the issue!
